.github/workflows/build_lint_test.yml

```
@@ -2,9 +2,9 @@ name: "Python package: build, lint, and test"
 
 on:
   push:
-    branches: [ "main" ]
+    branches: [ "main", "tests" ]
   pull_request:
-    branches: [ "main" ]
+    branches: [ "main", "tests" ]
```

```suggestion
    branches: [ "main" ]
```

Suggest to use manual workflow dispatch when targeting branches other than `main`